generators/chipyard/src/main/resources/csrc/cospike.cc

```diff
@@ -96,6 +102,27 @@ extern "C" void cospike_cosim(long long int cycle,
     uint64_t default_boot_addr = 0x80000000;
     boot_addr_reg->store(0, 8, (const uint8_t*)(&default_boot_addr));
 
+	for (auto& mem : mems) {
+		if (mem.first == info->mem0_base) {
+      std::string path_name = "chipyard-cosim-" + std::to_string(getpid());
+			ssize_t mem_size = mem.second->size();
+      int shared_fd = shm_open(path_name.c_str(), O_EXCL | O_RDWR, 0600);
+      if (shared_fd < 0) {
+        std::perror("[mm_t] shm_open for backing storage failed");
```

Comment:
If backing memory is not provided by DRAMSim, the cosim shouldn't fail. It would be more convenient if it printed an error message, indicating that it would assume zeroed memory.